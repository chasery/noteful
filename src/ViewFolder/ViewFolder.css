.FolderListWrapper {
  flex: 1;
  margin-right: 1rem;
}

.FolderListWrapper > h2 {
  margin: 0 0 1rem;
}

.NoteListWrapper {
  flex: 3;
}
