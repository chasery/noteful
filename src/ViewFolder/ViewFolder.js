import React from "react";
import FolderList from "../FolderList/FolderList";
import NoteList from "../NoteList/NoteList";

export default function ViewFolder() {
  return (
    <>
      <FolderList />
      <NoteList />
    </>
  );
}
