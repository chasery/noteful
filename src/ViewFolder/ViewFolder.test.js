import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import ViewFolder from "./ViewFolder";

describe("ViewFolder component", () => {
  it("renders without error", () => {
    const wrapper = shallow(<ViewFolder />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
