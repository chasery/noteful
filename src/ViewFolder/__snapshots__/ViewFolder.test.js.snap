// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewFolder component renders without error 1`] = `
<Fragment>
  <withRouter(FolderList) />
  <withRouter(NoteList) />
</Fragment>
`;
